src/Core/Scheduler/Providers/WFS_Provider.js

```diff
@@ -98,7 +97,8 @@ WFS_Provider.prototype.getFeatures = function getFeatures(crs, tile, layer) {
     if (result.feature !== undefined) {
         return Promise.resolve(result);
     }
-    return Fetcher.json(url, layer.networkOptions).then(geojson => assignLayer(Feature2Mesh.convert(GeoJSON2Features.parse(crs, geojson, tile.extent)), layer));
+
+    return Fetcher.json(url, layer.networkOptions).then(geojson => assignLayer(layer.convert(GeoJSON2Features.parse(crs, geojson, tile.extent), layer.style), layer));
```

Comment:
You're making `layer.convert` mandatory: you should check for its existence in preprocessLayer or define a sensible default value.